### DISCUSSION TITLE
Github's student developer pack and distance studies

### DISCUSSION THREAD

**Discussion by kohlerdigital** [community/community#79101] [GitHub Education]
### Hi everyone,

I am registered at a github recognised university as distance learner and situated in a different country. They know about it, it's all fine. 

I'm stuck at the proving part of the verification, I have the documents that shows the date i'm supposed to finish, i have a document that shows i'm distance learner, my name is displayed but the verification doesnt work.

message error

> The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
> 
> You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

Moreover, an extra error line about fixing the billing name appears. However, the name does correspond.

> Please correct your [GitHub billing information]) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.

==> I believe there is an OCR issue with the pictures uploaded. What do i do?

**Comment by rhuthik** [comment]
Since you're a distance learner, ensure your documents explicitly state that, as GitHub's system is likely looking for this specific mention. If the OCR isn't reading your documents correctly, try resubmitting clearer, higher-resolution images where your name and distance learner status are easily legible. If these steps don't help, I'd recommend submitting a support ticket to GitHub Education, choosing the option that your school's information might be incomplete or incorrect. They should be able to sort this out and get your verification on track!

**Reply by kohlerdigital** [reply]
It is _explicitly_ stated distance learner and as of quality, it is a screenshot from the documents, can't be more crystal clear. I guess i'd need to send a ticket.

**Comment by AbhijithMR-403** [comment]
Did you find any solution
